config/routes.rb

```diff
@@ -31,6 +31,7 @@
 
   resources :exhibits, path: '/', only: [] do
     resource :dor_harvester, controller: :"dor_harvester", only: [:create, :update]
+    resource :bibliography_resources, only: [:create, :update]
```

Comment:
Probably less important, but wondering if we should put the route behind the feature flag as well.

Reply:
We could.. but I didn't think it was really necessary. Happy to make that change if you think we should.